populate_casgenes.py

```diff
@@ -1,4 +1,6 @@
-#!/usr/bin/env python3
+import os
+import numpy as np
+# !/usr/bin/env python3
```

Comment:
The shebang should stay at the top. Though the imports should not really be here before the docstring either.